incubator/kubevirt/templates/apiserver-deployment.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: extensions/v1beta1
```

Comment:
You should use `apps/vbeta2` for deployments and daemonsets.

Reply:
fixed